app/models/certificate_template_entry.rb

```diff
@@ -4,6 +4,8 @@ class CertificateTemplateEntry < ApplicationRecord
 
   enum align: %i[center left right justify].freeze
 
+  HEX_COLOR_REGEX = /\A#(?:[0-9a-fA-F]{3}){1,2}\z/.freeze
```

Comment:
How is the `?:` part used here?

Reply:
In this case, `?:` is used to create a non-capturing group. It is necessary to combine part of the expression (`[0-9a-fA-F]{3}`) and apply the repetition `{1,2}` to it without creating an extra capturing group in the match results.